packages/fskit.rb

```diff
@@ -3,38 +3,36 @@
 class Fskit < Package
   description 'Filesystem utility library and SDK'
   homepage 'https://github.com/jcnelson/fskit'
-  version 'dd9a8'
+  version 'dd9a8d'
   source_url 'https://github.com/jcnelson/fskit/archive/dd9a8dc844b81afc63e5067397786a32ecb66008.tar.gz'
   source_sha256 'fe682890ebab9226d65fc6ebfb8b3619c0d5a93e3161787cea9d01ad23d3a83a'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8-chromeos-x86_64.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/fskit-dd9a8d-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'b51b018c892a80340f4957b64df87ea55f1a5dbe7bd5dc574bd504b8cefc3429',
-     armv7l: 'b51b018c892a80340f4957b64df87ea55f1a5dbe7bd5dc574bd504b8cefc3429',
-       i686: '7463ea0b68a5b29ea3b4162ccade4556d380ed55ba6c391de74c9821b8156388',
-     x86_64: '7ed7cade6d97d596a894b024785d910fa378d11017fb83487ab8a43ac446a784',
+     x86_64: '22c2b4d3adb6a374f267983ff164a6ad63f9aa762f47921c99eebe01031438a6',
   })
 
   depends_on 'attr'
 
   def self.patch
-    # for x86_64 architecture, build library should be modified to #{CREW_PREFIX}/lib64
-    if ARCH == "x86_64"
-      puts "Patch build file for x86_64 ..."
-      `sed -i -- 's/lib/lib64/g' buildconf.mk` # replace lib by lib64
-    end
+    system "wget", "https://github.com/JL2210/patches/raw/master/fskit-dd9a8-xattr.patch"
```

Comment:
Should we add a sha256 checksum on the patch?

Reply:
No. Right now, I'm planning on improving it. It's kind of crude right now. It just totally removes the header file and replaces it with two others.

Reply:
That doesn't give me much confidence.  We should probably add the sha256 checksum just in case.

Reply:
For now, I'll use a hashed version and add a sha256sum.

Reply:
BTW, patch fails with something along the lines of `patch: **** Only garbage was found in the patch input.` if the patch is corrupted.

Reply:
Well, with a sha256 checksum, at least they would get a message that tells them to try again.  I'm afraid they may just think the patch is corrupt if it doesn't download cleanly.

Reply:
Anyway, with binaries we don't have to worry about any of these edge cases.

Reply:
Done.